### PR TITLE
Create mkdocs

### PR DESCRIPTION
**Pull request by ianyu93**
This PR utilizes `mkdocs` to create a site. 

It includes:
- `mkdocs.yaml` for configuration
- `docs.yaml` GitHub workflow for deployment to GitHub Pages
- `docs/index.md` as Homepage

### CODE REVIEW

**Review by paulovn — DISMISSED** [review]
nice!

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+site_name: PIISA
+theme:
+  name: material
+
+nav:
+  - Home: index.md
+  - Libraries: libraries.md
+  - Proposals: proposals.md
+  - Specifications: specs.md
+  - Usage: usage.md
```
Comment (omri374) [inline mkdocs.yml]: Consider adding a link to Github:

```yaml
extra:
  social:
    - icon: fontawesome/brands/github
      link: https://github.com/piisa/piisa
```

**Review by omri374 — DISMISSED** [review]
Thanks! Great addition

**Review by huu4ontocord — APPROVED** [review]
This looks great! We can improve over time!

**Pull request merged**